diagnostic.md

```diff
@@ -5,20 +5,20 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    If you had a bunch of photo albums and you wanted to have the ability to create a new one, you could generate album-form component to handle that.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    There is the component.js file where you define all your actions that you may want to have when interacting with the component and the template for displaying the data.
```

Comment:
You are right.  The `component.js` is also responsible for declaring HTML attributes, and other presentation or behavior, and the `template.hbs` is responsible for containing markup, binding data, and binding and dispatching actions.